study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilder.java

```diff
@@ -376,6 +379,20 @@ private void insertStudyDetails(Handle handle, long studyId) {
         }
     }
 
+    private void insertStudySupportedLanguages(Handle handle, long studyId) {
+        String studyGuid = handle.attach(JdbiUmbrellaStudy.class).findById(studyId).getGuid();
```

Comment:
We have the `StudyDto`. Can we pass that in instead of making another database query?

Reply:
Good point, fixed!

Reply:
Done.